packages/ezflow_extension/ezextension/ezflow/design/standard/templates/block/edit/edit.tpl

```diff
@@ -156,7 +156,9 @@
         {undef $loop_count}
     {/if}
     {if and( not( $is_dynamic ), not( $is_custom ) )}
+        {def $start_browse_node_id = first_set( ezini( $block.type, 'ManualBlockStartBrowseNode', 'block.ini' ), 2 )}
```

Comment:
I wouldn't default to "2", because this changes the behavior. Instead, only output the hidden input field if the INI value is set.

(eZContentBrowse has its own INI fallback.)

Technically this is also a BC break if you have an override of block/edit/edit.tpl already. You would lose the ManualBlockStartBrowseNode effect. I would have put the fallback ManualBlockStartBrowseNode logic into PHP:

```
if( $http->hasVariable( 'start-browse-node-id' ) )
...
elseif( $blockINI->hasVariable( $block->attribute( 'type' ), 'ManualBlockStartBrowseNode' ) )
...
```

... then the template just has a commented out hidden input field for a developer to use or not.